tools/push_api_gh_pages.sh

```diff
@@ -6,7 +6,7 @@ GH_REPO="@github.com/snapcore/snapcraft.git"
 
 FULL_REPO="https://$GH_TOKEN$GH_REPO"
 
-cd docs/api/_build/html
+cd tools/api/_build/html
```

Comment:
I'm not quite sure what this is, but the `_build/` dir doesn't exist in my branch. Are you creating it somewhere else?